src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -109,22 +106,32 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
       }
     }, [isEditing]);
 
+    const AnimatedTypography = motion(TypographyComponent);
+    const isOverflowing = useIsOverflowing({ ref: typographyRef, ignoreHeightOverflow: true });
+    const shouldAnimate = useMemo(() => isOverflowing && isEditing !== null, [isEditing, isOverflowing]);
+
+    useLayoutEffect(() => {
+      if (!typographyRef.current) {
+        return;
+      }
+      const { width } = typographyRef.current.getBoundingClientRect();
+      setInputWidth(width);
+    }, [inputValue]);
```

Comment:
This was called in the HiddenPlaceholder component that is no longer needed, because I calculate the width using the already-rendered typography, it's just hidden when you're in edit mode